src/mme-app/handlers/attach_initue.c

```diff
@@ -593,6 +594,8 @@ void
 handle_monitor_processing(void *message)
 {
         log_msg(LOG_INFO, "Monitor Message Received");
+	free(message);
```

Comment:
May be you want to remove this free & return.

Reply:
fixed